improv(entity): Various fixes and tweaks for the Tempest

Zepalesque opened the pull request:
This pull request fixes a few issues and makes a few changes to the Tempests:
- The tempests now properly die during daytime (was broken since the pull request making them extend Zephyr)
- Tempests now hover up and down and have cloudlike particles underneath them, as they did in GoTV
- Tempests now have glowing tails, as they were _intended_ to in GoTV

Here is a video showing the new tweaks to their appearance and animation:

https://github.com/The-Aether-Team/The-Aether-Genesis/assets/60141811/3168eba1-293c-4219-ae67-f6c2bdf46609


---

I agree to the Contributor License Agreement (CLA).

bconlon1 commented on src/main/java/com/aetherteam/aether_genesis/event/listeners/EntityListener.java:
```diff
@@ -11,15 +13,17 @@
+        LivingEntity entity = event.getEntity();
+        if(entity.getType() == AetherEntityTypes.ZEPHYR.get() && !entity.getLevel().isClientSide && entity.getLevel().isNight())
+            event.setSpawnCancelled(true);
+        if(entity.getType() == GenesisEntityTypes.TEMPEST.get() && !entity.getLevel().isClientSide && !entity.getLevel().isNight())
```
The tempest spawn handling should already be handled in the tempest class.

Zepalesque replied:
ah, ok

bconlon1 committed:
ci: Update config.yml

[ci skip]

bconlon1 committed:
ci: Update continue_config.yml

[ci skip]

bconlon1 commented on src/main/java/com/aetherteam/aether_genesis/client/renderer/entity/TempestTransparencyGlowLayer.java:
```diff
@@ -0,0 +1,45 @@
+        return LAYER_GLOW;
+    }
+
+    /*    @Override
```
does this code need to be here if its commented out?

Zepalesque replied:
ah oops, I thought I removed that

Zepalesque commented:
hmm, I just remembered something _someone_ said, I can't remember who, but the spawn egg colors are wrong, is that something that would be a good idea to change, or should I keep it as it is now?

bconlon1 reviewed (approved):
lgtm

Aether-Team-Bot committed:
Merge branch 'local/Zepalesque/tempest-tweaks' into merge/tempest-tweaks

Aether-Team-Bot committed:
chore: Trigger build

bconlon1 committed:
ci: Update config.yml

Aether-Team-Bot committed:
chore: Trigger build

Aether-Team-Bot committed:
Merge branch 'local/Zepalesque/tempest-tweaks' into merge/tempest-tweaks

Zepalesque committed:
improv(entity): Various fixes and tweaks for the Tempest

- Tempests now hover up and down slightly like they did in GoTV
- Tempests now properly are killed at day, which was broken from the pull request making them extend the Zephyr class
- Tempests now have cloud-like particles underneath them as they did in GoTV
- Tempests have glowing tails, as they seemed to have been intended to have in GoTV (although it was somewhat flickery, which was likely a bug)
- The Tempest class file is better organized into com.aetherteam.aether_genesis.entity.monster.Tempest rather than com.aetherteam.aether_genesis.entity.Tempest

Zepalesque committed:
fix: removed redundant spawn cancel

Zepalesque committed:
docs: removed unneeded commented method

Zepalesque committed:
refactor: Consistent tempest spawn egg colors`
